### PR TITLE
Feat/manager 6742 - add vps status tile

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6742
| License          | BSD 3-Clause

## Description

Give to our customers status of their VPS 

### :flags: Translations
- [x] TRANS-35639

### CODE REVIEW

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js [hunk 1]
```diff
@@ -166,6 +166,24 @@ export default /* @ngInject */ ($stateProvider) => {
           servers: [server],
         });
       },
+
+      vpsStateInfo: /* @ngInject */ (stateVps) => {
```
Comment (radireddy) [inline packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js:170]: Not now but in the future, we can move this to vps.class and have methods like isError(), isSuccess(), and isWarning() etc.

**Review by radireddy — APPROVED** [review]

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js:170]: True, i thought like you.
There are a lot work to complete.
I will create a Jira to be sure we do it.

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js [hunk 2]
```diff
@@ -166,6 +166,24 @@ export default /* @ngInject */ ($stateProvider) => {
...
+          state,
+          stateGroup: {
+            error: ['maintenance', 'stopped', 'stopping'].includes(state),
+            warning: [
+              'backuping',
+              'installing',
+              'rebooting',
+              'upgrading',
+            ].includes(state),
+            success: ['running'].includes(state),
```
Comment (marie-j) [inline packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js:182]: I'm not sure this should be store in the routing, maybe a constant file and some testing or a utility function

Comment (mohammed-zahaf) [reply packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js]: Fixed.

**Review by frenautvh — DISMISSED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Comment by mohammed-zahaf** [comment]
Hello @antleblanc, this PR is ready to be merged and deployed.
Thanks.

**Comment by antleblanc** [comment]
Pulling translations

**Comment by antleblanc** [comment]
Translations has been pulled successfully.

**Comment by mohammed-zahaf** [comment]
Thanks @antleblanc to have pull trad

**Review by marie-j — APPROVED** [review]

**Pull request merged**